casper-cep78-js-client 1.3.0 (#207)
casper-ecosystem/enhanced-nft: client-js/CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.3.0] - 2023-04-03

### Changed

> - Added `casper-js-sdk` and `@make-software/ces-js-parser` as a peer dependencies to prevent conflicts between multiple versions of dependencies

Reviewer: ```suggestion
- Added `casper-js-sdk` and `@make-software/ces-js-parser` as peer dependencies. This will prevent conflicts between multiple versions of dependencies.
```

### Fixed

> - Fixed how `migrate` deploy is constructed.

Reviewer: ```suggestion
- Fixed how the `migrate` deploy is constructed.
```
